src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -16,24 +16,66 @@ public void GetAccessControl_DirectoryInfo_InvalidArguments()
         }
 
         [Fact]
+        public void GetAccessControl_DirectoryInfo()
+        {
+            DirectoryInfo directoryInfo = new DirectoryInfo("\\");
```

Comment:
These tests are Windows-only, but we could still use a temp directory here instead of the current one.